src/lightning/fabric/loggers/csv_logs.py

```diff
@@ -125,11 +125,13 @@ def experiment(self) -> "_ExperimentWriter":
         return self._experiment
 
     @rank_zero_only
-    def log_hyperparams(self, params: Union[Dict[str, Any], Namespace]) -> None:
+    def log_hyperparams(self, params: Union[Dict[str, Any], Namespace]) -> None:  # type: ignore[override]
```

Comment:
Why are we adding all these typing changes with this PR? How does this relate to the docs changes?

Reply:
`mypy` started to complain after using the latest Utils, and without fix I can't merge it so I could also bump it else were :)